### PR TITLE
Add support for Workers KV individual and bulk deletion API endpoints

### PR DESCRIPTION
**Pull request by gabbifish**
This PR adds support for the Workers KV [individual delete](https://api.cloudflare.com/#workers-kv-namespace-delete-key-value-pair) and [bulk deletion](https://api.cloudflare.com/#workers-kv-namespace-delete-multiple-key-value-pairs) API endpoints. It is used by https://github.com/cloudflare/wrangler/pull/452

### CODE REVIEW

File: src/endpoints/workerskv/delete_key.rs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+impl<'a> Endpoint<(), (), ()> for DeleteKey<'a> {
+    fn method(&self) -> Method {
+        Method::Delete
+    }
+    fn path(&self) -> String {
+        // Need multiple cases for presence of ttl or expiration parameters.
+        format!(
+            "accounts/{}/storage/kv/namespaces/{}/values/{}",
+            self.account_identifier, self.namespace_identifier, self.key
```
Comment (ashleymichal) [inline src/endpoints/workerskv/delete_key.rs]: url encode the key here (and then probably not in wrangler)

File: src/endpoints/workerskv/delete_bulk.rs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    pub namespace_identifier: &'a str,
+    pub bulk_keys: Vec<String>,
+}
+
+impl<'a> Endpoint<(), (), Vec<String>> for DeleteBulk<'a> {
+    fn method(&self) -> Method {
+        Method::Delete
+    }
+    fn path(&self) -> String {
+        // Need multiple cases for presence of ttl or expiration parameters.
```
Comment (adamchalmers) [inline src/endpoints/workerskv/delete_bulk.rs]: I don't understand this comment?

Comment (ashleymichal) [reply src/endpoints/workerskv/delete_bulk.rs]: same

Comment (gabbifish) [reply src/endpoints/workerskv/delete_bulk.rs]: Oop! This is an old comment. Removing.

File: src/endpoints/workerskv/delete_key.rs (hunk 1 above)
Comment (gabbifish) [reply src/endpoints/workerskv/delete_key.rs]: Done!

File: src/endpoints/workerskv/mod.rs [hunk 3]
```diff
@@ -30,3 +33,7 @@ pub struct Key {
 
 impl ApiResult for Key {}
 impl ApiResult for Vec<Key> {}
+
+pub fn url_encode_key(key: &str) -> String {
```
Comment (sssilver) [inline src/endpoints/workerskv/mod.rs]: Is `pub` necessary here?

Comment (gabbifish) [reply src/endpoints/workerskv/mod.rs]: great catch, I don't think so--this is only used in this module :)

Comment (gabbifish) [reply src/endpoints/workerskv/mod.rs]:
```suggestion
fn url_encode_key(key: &str) -> String {
```

**Pull request merged**